vllm/entrypoints/openai/api_server.py

```diff
@@ -672,8 +672,12 @@ async def reset_prefix_cache(raw_request: Request):
         Reset the prefix cache. Note that we currently do not check if the
         prefix cache is successfully reset in the API server.
         """
-        logger.info("Resetting prefix cache...")
-        await engine_client(raw_request).reset_prefix_cache()
+        device = None
+        device_str = raw_request.query_params.get("device")
+        if device_str is not None:
+            device = Device(device_str.upper())
```

Comment:
Please make v1 align with the same protocol, and error out when invalid device (i.e. cpu) is given.

Reply:
Done